manifests/testground/daemon/tg-daemon-deployment.yaml

```diff
@@ -48,16 +35,19 @@ spec:
             mountPath: "/go"
         resources:
           requests:
-            memory: 512Mi
+            memory: 700Mi
             cpu: 500m
           limits:
-            memory: 512Mi
+            memory: 1000Mi
+            cpu: 700m
       - name: testground-daemon
-        image: iptestground/testground:edge
+        #image: public.ecr.aws/n6b0k8i7/testground-daemon:80c5aca36114de067c33c8718cca95ef16db4c06
+        image: ghcr.io/celestiaorg/testground:39661ea
```

Comment:
we will need to bump to latest version in order to make all changes work

Reply:
updated to the latest available commit: `900c23e`


https://github.com/celestiaorg/testground/pkgs/container/testground/79647287?tag=900c23e